### ISSUE TITLE
Don't require absolute paths for cache hits when compiling Rust

### ISSUE THREAD

**Issue by Xanewok** [paritytech/cachepot#89]
The more broad issue is tracked upstream at https://github.com/mozilla/sccache/issues/35.

We're mostly interested in supporting only Rust for now and it seems that groundwork has been laid already via `--remap-path-prefix`. This issue should probably be addressed before we can deploy cachepot as the cache hit rate will be user-specific and thus fairly low initially.

**Comment by drahnr** [comment]
Related #37

**Comment by Xanewok** [comment]
FWIW basic support has been implemented via #116. Should we leave this open and track other missed cases and/or unless we can safely enable this feature by default? @drahnr @montekki

**Comment by drahnr** [comment]
I think we should split this into separate, mores specific issues:

* [ ] #124 
* [ ] #123
* [ ] ...

**Comment by drahnr** [comment]
Closing this, reference this issue as needed for follow-up issues :)
